datafusion/core/src/physical_plan/aggregates/mod.rs

```diff
@@ -579,6 +585,7 @@ impl AggregateExec {
         mut order_by_expr: Vec<Option<LexOrdering>>,
         input: Arc<dyn ExecutionPlan>,
         input_schema: SchemaRef,
+        limit: Option<usize>,
```

Comment:
Note: limit pushdown could be used even without the sort with min/max to reduce the final aggregation.

Reply:
Need to figure out how to fix:
```
error: this function has too many arguments (8/7)
```